images-list

```diff
@@ -110,6 +110,11 @@ quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.16.5
 quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.16.6
 quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.17.1
 quay.io/calico/pod2daemon-flexvol rancher/calico-pod2daemon-flexvol v3.17.2
+quay.io/cilium/cilium rancher/cilium-cilium v1.9.4
+quay.io/cilium/operator-aws rancher/cilium-operator-aws v1.9.4
+quay.io/cilium/operator-azure rancher/cilium-operator-azure v1.9.4
+quay.io/cilium/operator-generic rancher/cilium-operator-generic v1.9.4
+quay.io/cilium/startup-script rancher/cilium-startup-script 62bfbe88c17778aad7bef9fa57ff9e2d4a9ba0d8
```

Comment:
* @brandond can skopeo script handle image sha's ? It looks like it, but I don't think we have tried before. 
* None of these are created in dockerhub, you can DM me on how to get those created. But that is one of the checkboxes that needs to be checked before this can be merged. 
* Need a real issue created somewhere for purpose of change. You can put this in rancher/rancher if you don't have a place.

Reply:
@cbron I don't think it will handle a sha256 properly, but I can test out-of-band if you want to wait a bit...

Reply:
Oh I see, this isn't actually a sha256 blob hash, it's just a tag that corresponds to a git commit hash. That's fine.